src/Oro/Bundle/SecurityBundle/Acl/Domain/ObjectIdAccessor.php

```diff
@@ -4,12 +4,23 @@
 
 use Symfony\Component\Security\Acl\Model\DomainObjectInterface;
 use Symfony\Component\Security\Acl\Exception\InvalidDomainObjectException;
+use Symfony\Component\Security\Acl\Voter\FieldVote;
+
+use Oro\Bundle\EntityBundle\ORM\DoctrineHelper;
 
 /**
  * This class allows to get the class of a domain object
  */
 class ObjectIdAccessor
 {
+    /** @var DoctrineHelper */
+    protected $doctrineHelper;
+
+    public function __construct(DoctrineHelper $doctrineHelper)
```

Comment:
There are 5 DoctrineHelper classes in few bundles and all of them have different methods, the only common methods are: getEntityManager and getEntityRepository,
while getSingleEntityIdentifier method exists only in EntityBundle's DoctrineHelper